pom.xml

```diff
@@ -598,6 +598,41 @@
                 </plugins>
             </build>
         </profile>
+        <profile>
+            <id>windows-jenkins</id>
```

Comment:
Profiles popping up like mushrooms after the rain... Is it possible to specify the tests to ignore via some system variable or something that doesn't require separate profiles for everything?

Reply:
Ok, I'll try to refactor it to rather use `ignore.in.jenkins.on.windows` system property and run it in `all` profile.

Reply:
@jbartok Unfortunately using `Assume` based on system property and skipping test in `@BeforeClass` is problematic for this case. Containers in some classes (`com.hazelcast.jet.impl.connector.JmsSourceIntegrationTest_RabbitMQ`, `com.hazelcast.jet.impl.connector.WriteJdbcPTest` and `com.hazelcast.jet.s3.S3MockTest`) are defined in `@ClassRule` which means they are initialized before we can skip them (i.e. those tests they fail before they should be skipped).

I will use original implementation with `windows-jenkins` profile.

Reply:
Ok